core/server/common/src/main/java/alluxio/master/journal/raft/BufferedJournalApplier.java

```diff
@@ -283,12 +283,13 @@ protected void runCatchup() {
             try {
               mSuspendBuffer.wait();
             } catch (InterruptedException e) {
+              Thread.currentThread().interrupt();
```

Comment:
This was a bug? Was this discovered by one of the new tests?

Reply:
This is to silence sonarqube actually as it complained.
There is not much interrupting the thread that is going to terminate as soon as quitting this loop.